README.rst

```diff
@@ -1351,6 +1351,13 @@ Choose from the list of available rules:
 
   *Risky rule: this fixer may change functions named ``setUp()`` or ``tearDown()`` outside of PHPUnit tests, when a class is wrongly seen as a PHPUnit test.*
 
+* **php_unit_short_will_return**
```

Comment:
those are not methods of PHPUnit, but PHPUnit Mock. Please reflect that in name.

Reply:
Done